apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx

```diff
@@ -4,38 +4,30 @@ import get from 'lodash/get'
 import pick from 'lodash/pick'
 import { useIntl } from '@core/next/intl'
 
-import { Button } from '@condo/domains/common/components/Button'
+import {
+    DeleteButtonWithConfirmModal,
+    IDeleteActionButtonWithConfirmModal,
+} from '@condo/domains/common/components/DeleteButtonWithConfirmModal'
 
 import { BaseMeterModalForm } from '../components/BaseMeterModal/BaseMeterModalForm'
 import { Meter } from '../utils/clientSchema'
 import { IMeterUIState } from '../utils/clientSchema/Meter'
 
-const DeleteMeterButton = ({ meter, updateMeterAction }) => {
-    const intl = useIntl()
-    const DeleteMessage = intl.formatMessage({ id: 'Delete' })
-
-    const handleDeleteButtonClick = useCallback(() => updateMeterAction({ deletedAt: new Date().toDateString() }, meter),
-        [meter, updateMeterAction])
-
-    return (
-        <Button
-            type={'text'}
-            danger
-            onClick={handleDeleteButtonClick}
-        >
-            {DeleteMessage}
-        </Button>
-    )
-}
-
 const INITIAL_VALUES_KEYS = [
     'accountNumber', 'number', 'resource', 'place', 'numberOfTariffs', 'installationDate',
     'commissioningDate', 'sealingDate', 'verificationDate', 'nextVerificationDate', 'controlReadingsDate',
 ]
 
+const DELETE_BUTTON_CUSTOM_PROPS: IDeleteActionButtonWithConfirmModal['buttonCustomProps'] = {
```

Comment:
Kinda new syntax here? Was it confirmed?

Reply:
used `CustomButtonProps` instead